packages/backend/server/utils/markdown.ts

```diff
@@ -91,7 +91,7 @@ ${templatesStr}
 
 function generateTemplatesStr(templates: Record<string, string>) {
   const entries = Object.entries(templates).filter(([k]) => k !== "default");
-  let str = `[Open in Stackblitz](${templates["default"]})`;
+  let str = templates["default"] ? `[Open in Stackblitz](${templates["default"]})` : '';
 
   if (entries.length && entries.length <= 2) {
     str += ` • ${entries.map(([k, v]) => `[${k}](${v})`).join(" • ")}`;
```

Comment:
Can we remove the first dot if the default template is not there.

Reply:
Will there be a case when default template is not there but other templates are?